app/formations/mine/page.tsx

```diff
@@ -8,6 +8,7 @@ import { getUser } from "@/lib/server/users";
 
```

Comment:
I wonder if we need this `/mine` route at all now with the profile cards as mine = your user page? @KMontag42 thoughts?
 If you like the concept of having a generic URL it could just redirect to your user page if logged in

Reply:
I like the formations/mine page. I think the profile page may evolve into something more. Might just be temporary.